### ISSUE TITLE
Add small thumbnail left of sketch name in the app-navigation

### ISSUE THREAD

**Issue by jancborchardt** [ChristophWurst/sketch#22]
Similar to how the News app sidebar has favicons for the feeds, we should add small thumbnails in the left sidebar. This would make it much easier to discern the sketches.

Initially the note name can just be indented. Then once there was work on a sketch, it can simply be displayed on the left of the name. The background should be transparent instead of white so it blends in nicely.

**Comment by ChristophWurst** [comment]
Good point. For this feature some kind of thumbnail generator has to be created on the server – the client doesn't have all the needed information (e.g. lines).

**Comment by ChristophWurst** [comment]
Btw, thank you for your feedback. I appreciate that a lot :-)

**Comment by jancborchardt** [comment]
That’s what I’m here for. ;)

Maybe @georgehrke or @PVince81 can help regarding thumbnail generation?
